### ISSUE TITLE
Import "go to definition" not working for specific import value

### ISSUE THREAD

**Issue by niksy** [moetelo/twiggy#49]
When trying to "go to definition" for specific import, nothing happens, but the same functionality works when using command on import path.

For example, I would expect this would work when running command on `style_asset`, but this only works when running command on `"helpers.html.twig"`.

`base.html.twig`
![Screenshot 2024-11-08 at 14 28 53](https://github.com/user-attachments/assets/3e85b2f8-9931-4a03-a0bb-dd963e2fb475)

`helpers.html.twig`
![Screenshot 2024-11-08 at 14 29 04](https://github.com/user-attachments/assets/11b11431-f722-4862-b6b7-c203e92b9c70)

---

Running default Symfony project, with Sublime Text custom LSP client for Twiggy.

This is maybe related to language server specifically. I suppose this works for VS Code extension? Maybe it would be best if this functionality gets implemented in language server so every other editor can benefit from this.

**Comment by moetelo** [comment]
Yeah, this is related to the language server. VS Code part basically consists of nothing but `run server for each of the open folders` and `add space character when you type '{{'`
